src/App.js

```diff
@@ -1,16 +1,26 @@
-import Affiche from './components/ZoumCalculator';
-import { Nav, DisplayQuote } from './components/quote';
-import './components/Calculator.css';
+import './App.css';
+import React from 'react';
+import { Route, Routes } from 'react-router-dom';
+import Calculator from './components/MyCalculator';
+import Navigation from './components/Nav';
+import Home from './components/Home';
+import Quote from './components/Quote';
+import Footer from './components/Footer';
 
 function App() {
   return (
-
-    <div className="main">
-      <Nav />
-      <DisplayQuote />
-      <Affiche />
+    <div className="App">
+      <div className="wrapper">
+        <Navigation />
+        <Routes>
+          <Route path="/" element={<Home />} />
+          <Route path="/calculator" element={<Calculator />} />
+          <Route path="/quote" element={<Quote />} />
+        </Routes>
+        <Footer />
+      </div>
```

Comment:
- [OPTIONAL] - I would suggest that you adjust the alignments and overall look of your design to match the wireframe to improve the UI design☺ 

-  Creating a visually appealing design can help your app stand out from the competition.

Wireframe
![image](https://github.com/sam2if/Maths_magicians/assets/78251927/2018aea8-b30e-44f2-9abf-05c6fe3379b2)


Your design

- I would suggest the content cover the entire page similar to the wireframe

![image](https://github.com/sam2if/Maths_magicians/assets/78251927/68308290-56ab-47e4-b991-da4b7bd40586)

- Please adjust the position of the contents on the page similar to the wireframe 🛠

![image](https://github.com/sam2if/Maths_magicians/assets/78251927/8a9603c1-179b-473c-a86e-b138263cd3f2)